f34-py39/requirements.txt

```
@@ -602,50 +551,50 @@ ruamel.yaml.clib==0.2.6; platform_python_implementation == 'CPython' and python_
     --hash=sha256:d67f273097c368265a7b81e152e07fb90ed395df6e552b9fa858c6d2c9f42502 \
     --hash=sha256:dc6a613d6c74eef5a14a214d433d06291526145431c3b964f5e16529b1842bed \
     --hash=sha256:de9c6b8a1ba52919ae919f3ae96abb72b994dd0350226e28f3686cb4f142165c
-semantic-version==2.8.5; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3' \
-    --hash=sha256:45e4b32ee9d6d70ba5f440ec8cc5221074c7f4b0e8918bdab748cc37912440a9 \
-    --hash=sha256:d2cb2de0558762934679b9a104e82eca7af448c9f4974d1f3eeccff651df8a54
-sentry-sdk==1.5.0 \
-    --hash=sha256:0db297ab32e095705c20f742c3a5dac62fe15c4318681884053d0898e5abb2f6 \
-    --hash=sha256:789a11a87ca02491896e121efdd64e8fd93327b69e8f2f7d42f03e2569648e88
+semantic-version==2.9.0; python_version >= '2.7' \
+    --hash=sha256:abf54873553e5e07a6fd4d5f653b781f5ae41297a493666b59dcf214006a12b2 \
+    --hash=sha256:db2504ab37902dd2c9876ece53567aa43a5b2a417fbe188097b2048fff46da3d
+sentry-sdk==1.5.8 \
+    --hash=sha256:32af1a57954576709242beb8c373b3dbde346ac6bd616921def29d68846fb8c3 \
+    --hash=sha256:38fd16a92b5ef94203db3ece10e03bdaa291481dd7e00e77a148aa0302267d47
+setuptools==60.10.0; python_version >= '3.7' \
+    --hash=sha256:6599055eeb23bfef457d5605d33a4d68804266e6cb430b0fb12417c5efeae36c \
+    --hash=sha256:782ef48d58982ddb49920c11a0c5c9c0b02e7d7d1c2ad0aa44e1a1e133051c96
 six==1.16.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2' \
     --hash=sha256:1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926 \
     --hash=sha256:8abb2f1d86890a2dfb989f9a77cfcfd3e47c2a354b01111771326f8aa26e0254
 termcolor==1.1.0 \
     --hash=sha256:1d6d69ce66211143803fbc56652b41d73b4a400a2891d7bf7a1cdf4c02de613b
-thamos==1.21.1 \
-    --hash=sha256:38909ebb6345d98883fe03fd66e5204c014fafedff177662b6744b0c971467c9 \
-    --hash=sha256:b14907c7ead2a9bbeca0e80dc9a40d3c2152d7886ec5bbe9d605535cbdbf474a
+thamos==1.27.4 \
+    --hash=sha256:71fe3fd6660b996c1276425ffad09dc9f1f2d608a34f75507019805aa77f1b5d \
+    --hash=sha256:98cc133c1977afca29ffc76314e4baacb6fa2463f3d607ca795bcb8bdbd311d5
```

These are the wrong hashes (The old hashes are for thamos v1.27.3). I have provided the correct ones here.
```suggestion
    --hash=sha256:8def2131b43f0a6fb157f80dfb277ebfce387155ffb7e3da86b12ba5eef2be44 \
    --hash=sha256:d5d69c900b8ca90ff2feb9b89d358abddfbfb2c580359966206b93253a3019a2
```
The first hash on line 569 is the `.whl` and line 570 is the `.tar.gz`, and the same order has been kept for my suggestion